swan-lake/development-tutorials/ballerina-persist/supported-data-stores.md

```diff
@@ -221,3 +222,72 @@ Follow the steps below to obtain API tokens for Google Sheets.
     3. For step 1 under the  `Select & authorize APIs` section, select the required scopes for the API you are going to use. 
     4. Proceed with the necessary Google account validation steps. 
     5. Click **Exchange authorization code for tokens** to obtain the refresh and access tokens.
+
+## Redis [Experimental]
+
+The Redis data store is a key-value-based in-memory data store. It is useful for storing unstructured data and retrieving them quickly. It is not the default data store for `bal persist`. Therefore, you need to explicitly specify the data store when initializing `bal persist` in your application as follows.
```

Comment:
```suggestion
The Redis data store is a key-value-based inmemory datastore useful for storing and retrieving unstructured data quickly. It is not the default data store for `bal persist`. Therefore, you need to explicitly specify the data store when initializing `bal persist` in your application as follows.
```

Reply:
I have fixed this in https://github.com/ballerina-platform/ballerina-dev-website/pull/8838/commits/30d2d23c3fff8d3e9873f7d41128ccf8d49eb002

Reply:
As in the Redis website https://redis.io/, It is `in-memory` data store. Shall we keep it as `in-memory`